### PR TITLE
Make this installable with pip

### PR DESCRIPTION
**Pull request by traverseda**
Just because I wanted an easier way to add this to my python path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Author: Christoph Schranz, 12.01.2016
 
 `python Tweaker.py -i demo_object.stl -vb`
 
+If you want to install is as a CLI app first install
+[pipx](https://pypa.github.io/pipx/) and then run
+
+    pipx install git+https://github.com/ChristophSchranz/Tweaker-3.git
```
Comment (ChristophSchranz) [inline README.md:16]: Hi @traverseda ,

Are you sure the command is really with `pipx`? 

`pipx` returned "Package cannot be a url", but running with `pip` worked as expected. 

Ubuntu 20.04 here.

Comment (ChristophSchranz) [reply README.md:16]: It is not clear for me why there is a need for `pipx`. I'll check in a commit that uses `pip` only to simplify the setup. Feel free to update if there is a reason for `pipx`.

**Review by ChristophSchranz — COMMENTED** [review]
I already prefer this cli-interface myself, thanks for this commit!

File: README.md (hunk 1 above)
Comment (traverseda) [reply README.md:16]: It seems to work for me. There's no need to use pipx, but it help keep things properly isolated. If you install a bunch of python commands with normal pip you're going to get weird conflicts the dependencies eventually. Pipx works fine for me, I'm using version `0.16.3`.

image

Comment (traverseda) [reply README.md:16]: Works for me, but I imagine the people who have a need for pipx already use pipx

**Pull request merged**